apps/prairielearn/src/pages/instructorIssues/instructorIssues.html.ts

```diff
@@ -408,20 +408,20 @@ function FilterHelpModal() {
           </tr>
         </tbody>
       </table>
-      <h4>Full-text search</h4>
+      <h2 class="h4">Full-text search</h2>
```

Comment:
Given that modals now use an `<h2>` for their title, an `<h3>` might be more appropriate here.

Reply:
Updated in [7d5e2bc](https://github.com/PrairieLearn/PrairieLearn/pull/10440/commits/7d5e2bc50e687b8411d196edb8040872f0b802dd).